ui/src/components/filter/permission.js

```diff
@@ -1,6 +1,10 @@
-export default (accessType, action, permissions) => {
-  if (permissions[accessType].includes(action)) {
-    return true;
-  }
-  return false;
-};
+// import { authorizer } from '../../authorizer';
```

Comment:
Uh?

Reply:
I've forgotten to remove this JSDoc and this import :D